optout/src/main/java/gov/cms/ab2d/optout/OptOutProcessorImpl.java

```diff
@@ -24,27 +23,31 @@
 public class OptOutProcessorImpl implements OptOutProcessor {
 
     private final S3Gateway s3Gateway;
-    private final ConsentRepository consentRepository;
-    private final ConsentConverterService  consentConverterService;
+    private final OptOutRepository optOutRepository;
+    private final OptOutConverterService optOutConverterService;
 
 
 
     @Override
     @Transactional
     public void process() {
+        log.info("import opt-out data - start ...");
 
-        final InputStreamReader inputStreamReader = s3Gateway.getOptOutFile();
```

Comment:
Moved the InputStreamReader initialization inside the `try-with-resources` block to ensure it gets auto-closed along with the BufferedReader